G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice.Tests/Commands/ExcelOpen/ShouldOpenExcel_WhenExcelCommandWasCalled.G1ANT

```diff
@@ -0,0 +1,7 @@
+﻿excel.open errorcall ExcelDidNotOpen
+
+excel.close
+
+procedure ExcelDidNotOpen
+    test ⊂false⊃
```

Comment:
`test false`
Don't use macros where you don't really need them. Anyway you catch the exception just to rethrow it, which doesn't make sense to me.